switchbot_ros/launch/switchbot.launch

```diff
@@ -1,11 +1,13 @@
 <launch>
   <arg name="token" />
+  <arg name="secret" default="''" />
```

Comment:
you can use `null` and it will be treated as `None` in python.
```suggestion
  <arg name="secret" default="null"/>
```

Reply:
In my environments, both `<arg name="secret" default="null" />` and `<arg name="secret" default="" />` make an error `load_parameters: unable to set parameters (last param was [/switchbot_ros/secret=None]): cannot marshal None unless allow_none is enabled` and quit the process. But `<arg name="secret" default="''" />` works finely.

- My Environments
  - Ubuntu 20.04.6 LTS - ROS Noetic - Python 3.8.1
  - Ubuntu 18.04.6 LTS - ROS Melodic - Python 2.7.17

Does this problem depend on just only my environments?
Or do you know any solutions for the load_parameter error other than `<arg name="secret" default="''" />`?

---

The console output with load_parameter error occurs when I used `<arg name="secret" default="null" />` and `<arg name="secret" default="" />` in switchbot.launch is shown as followings.
``` bash
robotuser@robotuser-PC:~/switchbot_ws$ roslaunch switchbot_ros switchbot.launch token:='（省略）' --screen
... logging to /home/robotuser/.ros/log/fc70881c-34c9-11ee-84c2-43599b20a8be/roslaunch-robotuser-PC-68523.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://robotuser-PC:34991/

SUMMARY
========

PARAMETERS
 * /rosdistro: noetic
 * /rosversion: 1.16.0
 * /switchbot_ros/secret: None
 * /switchbot_ros/token: ca6f439a05b820c78...

NODES
  /
    switchbot_ros (switchbot_ros/switchbot_ros_server.py)

auto-starting new master
process[master]: started with pid [68531]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to fc70881c-34c9-11ee-84c2-43599b20a8be
process[rosout-1]: started with pid [68541]
started core service [/rosout]
load_parameters: unable to set parameters (last param was [/switchbot_ros/secret=None]): cannot marshal None unless allow_none is enabled
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/__init__.py", line 347, in main
    p.start()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 316, in start
    self.runner.launch()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/launch.py", line 676, in launch
    self._setup()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/launch.py", line 663, in _setup
    self._load_parameters()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/launch.py", line 354, in _load_parameters
    r  = param_server_multi()
  File "/usr/lib/python3.8/xmlrpc/client.py", line 879, in __call__
    return MultiCallIterator(self.__server.system.multicall(marshalled_list))
  File "/usr/lib/python3.8/xmlrpc/client.py", line 1109, in __call__
    return self.__send(self.__name, args)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 1447, in __request
    request = dumps(params, methodname, encoding=self.__encoding,
  File "/usr/lib/python3.8/xmlrpc/client.py", line 968, in dumps
    data = m.dumps(params)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 501, in dumps
    dump(v, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 523, in __dump
    f(self, value, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 576, in dump_array
    dump(v, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 523, in __dump
    f(self, value, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 594, in dump_struct
    dump(v, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 523, in __dump
    f(self, value, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 576, in dump_array
    dump(v, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 523, in __dump
    f(self, value, write)
  File "/usr/lib/python3.8/xmlrpc/client.py", line 527, in dump_nil
    raise TypeError("cannot marshal None unless allow_none is enabled")
TypeError: cannot marshal None unless allow_none is enabled
[rosout-1] killing on exit
[master] killing on exit
robotuser@robotuser-PC:~/switchbot_ws$ ^C
```